models/student.model.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+
+const studentSchema = mongoose.Schema(
+  {
+    name: { type: String, required: [true, 'Name is required'] },
+    email: {
+      type: String,
+      match: /.+\@.+\..+/,
+      required: [true, 'Email is required'],
+      validate: {
+        validator: async function (value) {
```

Comment:
Please view comments on login PR #3

Reply:
Hi Leonel, just added some logic to fix this.